fhir-swagger-generator/src/main/java/com/ibm/fhir/openapi/generator/FHIROpenApiGenerator.java

```diff
@@ -161,6 +165,8 @@ public static void main(String[] args) throws Exception {
         generateMetadataOpenApi();
         generateBatchTransactionOpenApi(filter);
         generateWholeSystemHistoryOpenApi();
+        if (filter.acceptOperation("export"))
+            generateExportOpenApi();
```

Comment:
we always use explicit blocks in our project
```suggestion
        if (filter.acceptOperation("export")) {
            generateExportOpenApi();
        }
```